lib/auth/contexts/whoami.js

```diff
@@ -3,19 +3,21 @@ const debug = require('debug')('codefresh:sdk:auth:whoami');
 
 const { Http } = require('../../../helpers/http');
 
-const http = Http({
-    timeout: 2000,
-    retryStrategy: () => false,
-});
+const http = Http();
```

Comment:
move this into the func- try to not have global variables

Reply:
it's ok for this place, it needs to be initialized only once. Or anyway move?